hoplite-core/src/main/kotlin/com/sksamuel/hoplite/ConfigFailure.kt

```diff
@@ -96,7 +96,7 @@ sealed interface ConfigFailure {
 
   data class UnresolvedSubstitution(val value: String, val node: Node) : ConfigFailure {
     override fun description(): String {
-      return "Resolved substitution $value at ${node.pos.loc()}"
+      return "Unresolved substitution $value at ${node.pos.loc()}"
```

Comment:
I guess this change should also update the test at https://github.com/sksamuel/hoplite/blob/e33c55aeaab773e290f701247a116e34f5912d70/hoplite-yaml/src/test/kotlin/com/sksamuel/hoplite/yaml/EnvVarPreprocessorTest.kt#L58, or?

Reply:
Yep thanks.